jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/modal/ModalVerticallyCenteredTests.java

```diff
@@ -0,0 +1,171 @@
+                        modalVerticallyCentered.modal2
+                }
+        };
+    }
+
+    @DataProvider
+    public Object[][] modalCssData() {
+        return new Object[][]{
+                {
+                        modalVerticallyCentered.modalCenterTrigger,
+                        modalVerticallyCentered.dismissModal1Close,
+                        modalVerticallyCentered.modal1,
+                        "modal-dialog-centered"
+                },
+
+                {
+                        modalVerticallyCentered.modalCenterScrollableTrigger,
+                        modalVerticallyCentered.dismissModal2Close,
+                        modalVerticallyCentered.modal2,
+                        "modal-dialog-centered"
+                }
+        };
+    }
+
+    @DataProvider
+    public Object[][] modalVerticalAlignmentData() {
+        return new Object[][]{
+                {
+                        modalVerticallyCentered.modalCenterTrigger,
+                        modalVerticallyCentered.dismissModal1Close,
+                        "button:nth-of-type(1)",
+                        "modal-vertical-content-1"
+                },
+
+                {
+                        modalVerticallyCentered.modalCenterScrollableTrigger,
+                        modalVerticallyCentered.dismissModal2Close,
+                        "button:nth-of-type(2)",
+                        "modal-vertical-content-2"
+                }
+        };
+    }
+
+    @DataProvider
+    public Object[][] modalDismissData() {
+        return new Object[][]{
+                {
+                        modalVerticallyCentered.modalCenterTrigger,
+                        modalVerticallyCentered.dismissModal1Close,
+                        modalVerticallyCentered.modal1
+                },
+
+                {
+                        modalVerticallyCentered.modalCenterScrollableTrigger,
+                        modalVerticallyCentered.dismissModal2Close,
+                        modalVerticallyCentered.modal2
+                }
+        };
+    }
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test(dataProvider = "modalBasicData")
+    public void modalBasicFunctionalityTest(Button showButton,
+                                            Modal modal) {
+        showButton.show();
+        showButton.click();
+
+        modal.core().waitFor().displayed();
+
+        modal.is().displayed();
+
+        modal.close();
+
+        modal.core().waitFor().hidden();
+
+        modal.is().hidden();
+    }
+
+    @Test(dataProvider = "modalCssData")
+    public void modalCssTest(Button showButton,
+                             Button dismissButton,
+                             Modal modal,
+                             String modalBgCss) {
+        showButton.show();
+        showButton.click();
+
+        modal.childs().get(0).is().hasClass(modalBgCss);
```

Comment:
we need a task to fix childs method, so it would be great if you up that topic during meeting

Reply:
Ok

Reply:
It self-resolved and now works as expected with get(1), get(2) etc...